data/analytics/attributes.yml

```diff
@@ -76,10 +76,10 @@
   redact: false
 
 - name: index
```

Comment:
Do we need data for the `index` attribute now? It's only a container for sub attributes. We don't have one for `event_data`.

Reply:
Sure, I can remove it, but if I do it will break the `print this page` and `share this page` pages, as they are using the defaults for `index` (i.e. they aren't setting the value themselves), so without it, the whole `index` value disappears on them. Is this OK?

Reply:
Oh of course, thanks. Okay, we'll update those two and as part of that remove the `index` attribute as part of a separate PR, thanks.

Reply:
Starting to pull that PR together here: https://github.com/alphagov/govuk-developer-docs/pull/4002